app/workers/results_notification_worker.rb

```diff
@@ -24,7 +24,7 @@ def run_results_email!
 
   def send_results_email(user, time_period)
     word_counts = time_period.responses.completed.where.not(emotion_id: nil).includes(:emotion)
-                             .where('emotions.category' => ['positive', 'negative'])
+                             .where('emotions.category' => %w(positive negative))
                              .group('emotions.word', 'emotions.category')
```

Comment:
before Use %w or %W for an array of words. NEW
                             .where('emotions.category' => ['positive', 'negative'])
after %w-literals should be delimited by [ and ]. NEW
                             .where('emotions.category' => %w(positive negative))